node/compile-hook.js

```diff
@@ -5,14 +5,20 @@ const compiler = require("./caching-compiler.js");
 const Module = require("./runtime.js").Module;
 const Mp = Module.prototype;
 
+let overrideCompileOptions = {};
+
+module.exports = (options) => {
```

Comment:
Maybe also update the `exports` variable in this assignment, just in case we ever accidentally use that in this module.